tools/qemu_cloud_start

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This and qemu_cloud_stop are no longer needed?

Reply:
true. i just removed them